cert_agent/cert_agent/gunicorn_conf.py

```diff
@@ -0,0 +1,11 @@
+import logging
+import os
+import beeline
+
+
+def post_worker_init(worker):
+    logging.info(f'beeline initialization in process pid {os.getpid()}')
+    beeline.init(
+        writekey=os.environ.get('HONEYCOMB_WRITEKEY'),
+        dataset=os.environ.get('HONEYCOMB_DATASET'),
+        service_name='cert_agent')
```

Comment:
I'm not sure if `beeline` would simply not initialize itself if no `writekey` is provided, may be it does. I've added this suggestion anyway.

```suggestion
if os.environ.get('HONEYCOMB_WRITEKEY'):
        beeline.init(
            writekey=os.environ['HONEYCOMB_WRITEKEY'],
            dataset=os.environ['HONEYCOMB_DATASET'],
            service_name='cert_agent')
```